WooCommerce/Classes/POS/Presentation/Order Messages/PointOfSaleOrderSyncCouponsErrorMessageView.swift

```diff
@@ -0,0 +1,89 @@
+import SwiftUI
+
+@available(iOS 17.0, *)
+struct PointOfSaleOrderSyncCouponsErrorMessageView: View {
+    let message: String
+    let retryHandler: () -> Void
+
+    @Environment(PointOfSaleAggregateModel.self) private var posModel
+    @Environment(\.dynamicTypeSize) var dynamicTypeSize
+
+    private var attributedMessage: AttributedString {
+        if let data = message.data(using: .utf8),
+           let nsAttributedString = try? NSAttributedString(
+               data: data,
+               options: [.documentType: NSAttributedString.DocumentType.html],
+               documentAttributes: nil) {
+            var attributedString = AttributedString(nsAttributedString)
```

Comment:
TIL we can also attempt to parse this to markdown via `try AttributedString(markdown: message)` (need to import Foundation). Maybe would be cleaner than using NSAttributedString, or we could keep both as a fallback if the initial decoding fails.

Reply:
Thanks! I'll try 👍

Reply:
Unfortunately, it doesn't parse through all the HTML tags. I tried different options but it didn't work.